src/bots/github/GitHubAPIMistralLargeBot.js

```
@@ -0,0 +1,11 @@
+import GitHubAPIBot from "./GitHubAPIBot";
+
+export default class GitHubAPIMistralLargeBot extends GitHubAPIBot {
+  static _className = "GitHubAPIMistralLargeBot";
+  static _model = "Mistral-large-2407";
+  static _logoFilename = "mistral-large-github-logo.png";
+
+  constructor() {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)